### PR TITLE
osd/osdmonitor: pool of objects beyond quota according to warn, when …

### PR DESCRIPTION
**Pull request by renhwztetecs**
…the pool bytes beyond set quotas

pool of objects beyond quota according to warn, when the pool bytes beyond set quotas

Signed-off-by: huanwen ren <ren.huanwen@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
The original procedure sounds more reasonable to me as it's really a rare case that we hit the both limitations of bytes and objects.

File: src/mon/OSDMonitor.cc [hunk 1]
```diff
@@ -4065,14 +4065,14 @@ bool OSDMonitor::update_pools_status()
...
+      } 
+      if (pool.quota_max_objects > 0 &&
 		 (uint64_t)sum.num_objects >= pool.quota_max_objects) {
         mon->clog->warn() << "pool '" << pool_name << "' is full"
                          << " (reached quota's max_objects: "
                          << pool.quota_max_objects << ")";
-      } else {
-        assert(0 == "we shouldn't reach this");
       }
+
```
Comment (athanatos) [inline src/mon/OSDMonitor.cc]: Trailing whitespace.

File: src/mon/OSDMonitor.cc [hunk 2]
```diff
@@ -4065,14 +4065,14 @@ bool OSDMonitor::update_pools_status()
         mon->clog->warn() << "pool '" << pool_name << "' is full"
                          << " (reached quota's max_bytes: "
                          << si_t(pool.quota_max_bytes) << ")";
-      } else if (pool.quota_max_objects > 0 &&
+      }
```
Comment (athanatos) [inline src/mon/OSDMonitor.cc]: and here

**Comment by athanatos** [comment]
@renhwztetecs  I think this makes sense, reopen once you've fixed up the whitespace issues!

**Comment by renhwztetecs** [comment]
@athanatos 
sorry i'm later
I change fixed up the whitespace issues, whether can help reopen
thanks

**Comment by renhwztetecs** [comment]
@athanatos 
I'm later
I change fixed up the whitespace issues, whether can help reopen
thanks

**Comment by athanatos** [comment]
@renhwztetecs Did you reopen it?  It seems to suggest that you opened a new one, but it doesn't actually give me the link.

**Comment by renhwztetecs** [comment]
@athanatos 
I'm later
I change fixed up the whitespace issues, whether can help reopen
thanks

**Pull request closed without merging**